whipper/common/encode.py

```diff
@@ -89,3 +90,71 @@ def _tag(self):
         w.save()
 
         self.stop()
+
+
+class EmbedPictureTask(task.Task):
+    description = 'Embed picture to FLAC'
+
+    def __init__(self, track_path, coverart_path):
+        self.track_path = track_path
+        self.coverart_path = coverart_path
+
+    def start(self, runner):
+        task.Task.start(self, runner)
+        self.schedule(0.0, self._embed_picture)
+
+    def _make_flac_picture(self, coverart_filename):
```

Comment:
Let’s be consistent:
```suggestion
    def _make_flac_picture(self, cover_art_filename):
```
(and `s/coverart_filename/cover_art_filename/g`)